### DISCUSSION TITLE
👋 We want to hear from you!

### DISCUSSION THREAD

**Discussion by SmallLemur** [community/community#10119] [Copilot Conversations]
:wave: Hi everyone,
We’re looking for any and all feedback to help make Copilot better. If you’ve been exploring how Copilot can help you in your development workflow, we’d love to hear from you.

[Book a time to talk with us!](https://calendly.com/smalllemur/60min)

**Comment by Mafiajakudza12** [comment]
Лемуррр

**Reply by d7mtg** [reply]
Calendly link no longer works...

**Comment by pEGgysue22** [comment]
> 👋 Hi everyone, We’re looking for any and all feedback to help make Copilot better. If you’ve been exploring how Copilot can help you in your development workflow, we’d love to hear from you.
> 
> [Book a time to talk with us!](https://calendly.com/smalllemur/60min)

**Comment by estrng** [comment]
Ok let's do it!

**Reply by Leninmathan** [reply]
Hi

**Comment by MDR3EM** [comment]
☺️

**Comment by Prasetyaak** [comment]
🤲

**Comment by godarren** [comment]
> 👋 Hi everyone, We’re looking for any and all feedback to help make Copilot better. If you’ve been exploring how Copilot can help you in your development workflow, we’d love to hear from you.
> 
> [Book a time to talk with us!](https://calendly.com/smalllemur/60min)

This is awesome and I would gladly be the subject of a product discussion sometime soon, but I have one important request that I want to suggest right away while I'm thinking about it if you don't mind. I've been coming across a situation a lot where I would love to accept the first line of a suggestion but not the rest of the content. I feel like there should be (maybe is?) a keyboard shortcut for this...

Sometimes the only suggestions are long blocks of code and so rather than have to remove all of that mess after accepting the solution, I end up having to type the line as I would have normally. This happens with things like loops or switch case conditionals. It's times where I know if I get inside the block then start typing the first line copilot is going to realize what I actually want to do, so just getting past that first line sometimes seems like more of a chore than it would be if I could easily accept the current line rather than the whole code block.

Edit: I found this related issue just now, thanks for raising it @SmallLemur, for anyone else interested in following:
https://github.com/microsoft/vscode/issues/140708

**Reply by SmallLemur** [reply]
@godarren thank you for the suggestion :) we are working our way through a long list of ticket at the moment. I would be happy to chat with you, feel free to book a meeting !
Cheers !

**Reply by baruchsheffi** [reply]
Had the same issue while copilot was working with me, it/he suggested to finish my array with 3 right items and 4 wrong ones. needed to accept everything and start deleting...
I felt copilot is my new bestie!! but sadly, it stopped working (wrote about it in another feedback)

Thank god I didn't throw my rubber duck just yet...

**Comment by Allen191819** [comment]
Hopefully, there will be better support for NVIM with lua.

**Comment by Tesserex** [comment]
Is one of your existing tickets to stop copilot from stomping on and removing intellisense when you type a period? That's my one big gripe so far. I type the period and just want to see methods and properties that actually exist, but instead I get copilot suggesting things that don't exist. No amount of escape key, retyping it, even the keyboard shortcut for suggestions ever gets intellisense to show up again.

**Comment by 0x2me** [comment]
Can i get an invite please ?

**Comment by d7mtg** [comment]
Any way to make it read code from below the part I'm working on?
Like I'm working in the `<style>` HTML tag, and it doesn't know any of my classes because I'm above the `body`.

**Comment by crackwitz** [comment]
I felt the need to express myself and this seemed the most appropriate thread without opening a new one... I'm a bit sad that all the others are various types of criticism and I don't think you get enough praise for this thing.

I am **absolutely thrilled** with Copilot. Been trying it out on a few small toy scripts (linear algebra, computer graphics/vision), now using it to write a GUI tool for some automation at our student council. It feels like pair programming with a somewhat passive but attentive and basically competent "driver". Definitely more sentient than a lot of the people popping up on Stack Overflow 😆 If this thing gains the ability to *be spoken to* (and know what I'm gesticulating at with the mouse pointer), to feed in architectural thoughts, intentions, directions, and maybe even *to ask me stuff*, that would blow my mind.
